apps/mobile/src/state/chat/atoms/sendTradeChecklistUpdateActionAtom.ts

```diff
@@ -64,6 +65,7 @@ export default function createSubmitChecklistUpdateActionAtom(
             },
             time: unixMillisecondsNow(),
             uuid: generateChatMessageId(),
+            myVersion: version,
```

Comment:
Good catch. It's not necessary, but it's good to have this here.

Reply:
Actually without this parameter, if one side approved the trade and right away sent trade checklist message (date, price, place of meet or other) and the other side received the message on the background, not with the chat detail open, it does not display the trade checklist messages as the app did not know the version from "last message received" and displayed the messages once received message that contained this attribute (lets say normal message). It was the bug that slush reported, that the other side did not receive his trade checklist updates